CSOAR-1861: Add csoar cloud execution egress topic (#3939)
SumoLogic/sumologic-documentation: docs/cloud-soar/architecture.md

Cloud SOAR offers a patent-pending Automated Responder Knowledge (DF-ARK) module which applies machine learning to historical responses and threats. It recommends relevant Playbooks, paths of action to expedite the process, and responses to manage and mitigate similar incidents with better response time.

> Cloud Soar provides static egress for Cloud Executions. IP addresses can be entered into the allowlist. For a list of cloud soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).

Reviewer: ```suggestion
Cloud Soar provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud Soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).
```

> Cloud Soar provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud Soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).

Reviewer 2: ```suggestion
Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/hc/en-us).
```

> Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/hc/en-us).

Reviewer: ```suggestion
Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/support/s/).
```
